rpms.in.yaml

```diff
@@ -0,0 +1,10 @@
+contentOrigin:
+  repofiles:
+    - rhocp.repo
+packages:
+  - openssl
```

Comment:
What rpm package do you actually want? Is it `openssl`?

Reply:
updated it as we speak, copypaste issues ;) the package I wanted to add was promu